src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line to follow the coding conventions.

Reply:
++